.github/workflows/build.yml

```diff
@@ -16,11 +16,11 @@ jobs:
     - name: Setup .NET Core
       uses: actions/setup-dotnet@v1
       with:
-        dotnet-version: 5.0.100-preview.7.20366.6
+        dotnet-version: 5.0.100-preview.8.20417.9
     - name: Test
-      run: dotnet test --verbosity normal
+      run: ./build test
     - name: Pack
-      run: dotnet pack --configuration Release --version-suffix build$GITHUB_RUN_NUMBER.0 -p:ContinuousIntegrationBuild=true
+      run: ./build.cmd pack
```

Comment:
```suggestion
      run: ./build.sh pack
```

Reply:
cmd vs sh doesn't make a difference in this context. the cmd file is setup to redirect.
NUKE recommends using the cmd file, since it works anywhere.

Reply:
Hrm not a fan as the CI runs on Ubuntu agents, but fine.